shared/src/metabase/shared/util/currency.cljc

```diff
@@ -634,6 +634,13 @@
           :rounding 0,
           :code "PEN",
           :name_plural "Peruvian nuevos soles"}],
+   [:PGK {:symbol "K",
+          :name "Papua New Guinean Kina",
+          :symbol_native "K",
+          :decimal_digits 2,
+          :rounding 0,
```

Comment:
there's no 1 or 2 toea anymore, to note - there's like 10 currencies where rounding in MB is 0 but rounding in real life is to the 5th or 10th subunit. I guess the justification is to chart trading and o&g and other stuff where people round to smaller decimals. we're just not consistent about this, I won't stop you